### ISSUE TITLE
Cannot create notebook through web UI

### ISSUE THREAD

**Issue by rangerbottle** [kubeflow/kubeflow#6572]
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
1. Install Kubeflow on local-cluster ( kind )
2. go to web UI and login with default user http://localhost:8080/_/jupyter/new?ns=kubeflow-user-example-com
3. Go to notebook panel and create a default one
4. After click "Launch", got error "[500] An error occured in the backend. http://localhost:8080/jupyter/api/namespaces/kubeflow-user-example-com/notebooks"
5. After check the pod "jupyter-web-app-deployment" log, found 

2022-07-13 09:20:24,419 | apps.common.form | INFO | Using provided value for 'workspace': {'mount': '/home/jovyan', 'newPvc': {'metadata': {'name': 'notebook1-volume'}, 'spec': {'accessModes': ['ReadWriteOnce'], 'resources': {'requests': {'storage': '10Gi'}}}}}
2022-07-13 09:20:24,419 | kubeflow.kubeflow.crud_backend.errors.handlers | ERROR | Caught and unhandled Exception!
2022-07-13 09:20:24,419 | kubeflow.kubeflow.crud_backend.errors.handlers | ERROR | create_notebook() got an unexpected keyword argument 'dry_run'
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/usr/local/lib/python3.7/site-packages/kubeflow/kubeflow/crud_backend/decorators.py", line 18, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/kubeflow/kubeflow/crud_backend/decorators.py", line 40, in runner
    return func(*args, **kwargs)
  File "/src/apps/default/routes/post.py", line 48, in post_pvc
    api.create_notebook(notebook, namespace, dry_run=True)
TypeError: create_notebook() got an unexpected keyword argument 'dry_run'

**What did you expect to happen:**
Create notebook server

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Kubeflow version: (version number can be found at the bottom left corner of the Kubeflow dashboard): build version dev_local (under left corner of UI dashboard)
- kfctl version: (use `kfctl version`):
- Kubernetes platform: (e.g. `minikube`) kind
- Kubernetes version: (use `kubectl version`):

-WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.2", GitCommit:"f66044f4361b9f1f96f0053dd46cb7dce5e990a8", GitTreeState:"clean", BuildDate:"2022-06-15T14:22:29Z", GoVersion:"go1.18.3", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.4
Server Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.4", GitCommit:"e6c093d87ea4cbb530a7b2ae91e54c0842d8308a", GitTreeState:"clean", BuildDate:"2022-03-06T21:32:53Z", GoVersion:"go1.17.7", Compiler:"gc", Platform:"linux/amd64"}

- OS (e.g. from `/etc/os-release`):
Ubuntu 18.04

**Comment by streamnsight** [comment]
same issue here

**Comment by streamnsight** [comment]
the repo has the requirements for Flask==1.1.1 and Flask-API==2.0
https://github.com/kubeflow/kubeflow/blob/v1.6.0-rc.0/components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/requirements.txt

but the image has
```
Flask              2.1.2
Flask-API          3.0.post1
```

**Comment by kimwnasptd** [comment]
Can confirm I see this error as well. Will take a look asap

cc @annajung

**Comment by mokpolar** [comment]
same issue here

jupyter-web-app image is docker.io/kubeflownotebookswg/jupyter-web-app:v1.6.0-rc.0

**Comment by kimwnasptd** [comment]
Next steps are:
1. Update the [`.dockerignore`] file to ignore the `build` folder of the common backend code
2. Rebuild all the images from the GH actions

For step 2 we'll create a new RC and the actions will build everything. So this will be resolved in the next RC.

**Comment by mokpolar** [comment]
@kimwnasptd it works! thanks!

**Comment by mokpolar** [comment]
@kimwnasptd  @streamnsight but, still can't find VSCode, R imageGroup when I click New Notebook 

Does this only happen to me?


<img width="642" alt="image" src="https://user-images.githubusercontent.com/19700890/179337807-aac6d589-cc39-44f4-9451-f2ee92bb4169.png">

**Comment by masoumi76** [comment]
I am facing the same issue. I have installed kubeflow via manifests and getting the same error "[500] An error occured in the backend". I'm not sure how to rebuild images via kubeflow manifests. Any help would be appreciated.

**Comment by majidhali** [comment]
Same issue here, installed kubeflow via manifests and everytime I try to create new notebook I got the below error:
[500] An error occured in the backend. http://localhost:63485/jupyter/api/namespaces/kubeflow-user-example-com/notebooks
I read an article here having the same issue (PodDefaults in the Namespace doesn't have a desc field):
https://github.com/kubeflow/kubeflow/issues/3770
but I have no idea how to change the desc field!

**Comment by jhasm** [comment]
Hi @kimwnasptd any instructions or pointers for building from source? What should we build from source and how?

**Comment by streamnsight** [comment]
jupyter server code is here: 
https://github.com/kubeflow/kubeflow/tree/master/components/example-notebook-servers
just go into jupyter folder, build the image and push it to your own repo, then update image in the manifests.

but this is 1.6RC0, so maybe you'd be better off using 1.5.1, if you want stable. pull the 1.5.1 manifests using the git tag v1.5.1

**Comment by kimwnasptd** [comment]
Can confirm the bug with showing only one image group. @haoxins this seems to be caused by https://github.com/kubeflow/kubeflow/pull/6482. I tried to revert the changes of the PR and then I could see the three image groups as expected.

Can you please take a look? This is currently a release blocker cc @annajung 

@masoumi76 @majidhali @jhasm in order to build the jupyter web app image you can use the rules we have in the Makefile https://github.com/kubeflow/kubeflow/blob/master/components/crud-web-apps/jupyter/Makefile

**Comment by haoxins** [comment]
> Can confirm the bug with showing only one image group. @haoxins this seems to be caused by #6482. I tried to revert the changes of the PR and then I could see the three image groups as expected.
> 
> Can you please take a look? This is currently a release blocker cc @annajung
> 
> @masoumi76 @majidhali @jhasm in order to build the jupyter web app image you can use the rules we have in the Makefile https://github.com/kubeflow/kubeflow/blob/master/components/crud-web-apps/jupyter/Makefile

will take a look

**Comment by haoxins** [comment]
@kimwnasptd https://github.com/kubeflow/kubeflow/pull/6596

**Comment by chengleqi** [comment]
> jupyter server code is here: https://github.com/kubeflow/kubeflow/tree/master/components/example-notebook-servers just go into jupyter folder, build the image and push it to your own repo, then update image in the manifests.
> 
> but this is 1.6RC0, so maybe you'd be better off using 1.5.1, if you want stable. pull the 1.5.1 manifests using the git tag v1.5.1

We can use v1.6.0-rc.1 now. It worked well.

**Comment by zijianjoy** [comment]
> Can confirm the bug with showing only one image group. @haoxins this seems to be caused by #6482. I tried to revert the changes of the PR and then I could see the three image groups as expected.

When fixing the image group items on Create Notebook page, please note that this is intended to show `1` and `2` instead of VSCode and Rstudio logos due to license issue: 
<img width="768" alt="Screen Shot 2022-08-16 at 8 15 41 AM" src="https://user-images.githubusercontent.com/37026441/184916362-1b40f746-7b39-422f-bed6-bb2bbd8326f1.png">

Reference: https://github.com/kubeflow/kubeflow/pull/5884

cc @kimwnasptd @annajung

**Comment by annajung** [comment]
Thanks @zijianjoy for clarification from the community meeting. The release blocking issue that the release team is tracking is that only one image group is available in the 1.6 RC and other options (1 and 2) are not showing up. This part has been labeled as the release-blocking issue via [comment](https://github.com/kubeflow/kubeflow/issues/6572#issuecomment-1194513442)

**Comment by jalola** [comment]
Hi @zijianjoy ,

I installed Kubeflow v1.6.0-rc.1 but the the 2 groups (code and rstudio) are not showed still.
The image of `jupyterwebapp` is `docker.io/kubeflownotebookswg/jupyter-web-app:v1.6.0-rc.1`

**Comment by kimwnasptd** [comment]
Hey everyone, this issue contained 2 problems so let me try to expose the final state.

The first issue was with getting 500 error on the LAUNCH button. This was a cache issue with how we built the image and was fixed in the latest RC https://github.com/kubeflow/kubeflow/issues/6572#issuecomment-1185740944.

The second one was with seeing only the option for the Jupyter notebook type in the form https://github.com/kubeflow/kubeflow/issues/6572#issuecomment-1186080541. We addressed this issue with https://github.com/kubeflow/kubeflow/pull/6625 and this will be fixed in the next RC we'll cut on Monday/Tuesday.

Also @zijianjoy regarding your point about the logos, yes we'll keep the `1` and `2` logos for the copyright situation. We have this in our mind.
